tests/test_core/test_pipeline.py

```diff
@@ -47,3 +50,14 @@ def test_pipeline_execution(simple_config_filename):
     folder_content = sorted(os.listdir(logs_folder))
     assert len(folder_content) == 5
     assert folder_content[0].startswith("eoexecution-report")
+
+
+def test_get_patch_list_filtration(simple_config_filename: str) -> None:
+    """Tests that the `test_subset` filtration is done correctly."""
+    config = interpret_config_from_path(simple_config_filename)
+    pipeline = SimplePipeline.from_raw_config(config)
+    expected_patch_list = [
+        ("eopatch-id-0-col-0-row-0", BBox(((729480.0, 4390045.0), (732120.0, 4391255.0)), crs=CRS(32638))),
+        ("eopatch-id-1-col-0-row-1", BBox(((729480.0, 4391145.0), (732120.0, 4392355.0)), crs=CRS(32638))),
+    ]
+    assert pipeline.get_patch_list() == expected_patch_list
```

Comment:
add empty test case 

i would also parametrize the test case for easier addition of new cases